components/NutrientTable/NutrientTable.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Nutrient } from '../../types/product';
+
+type NutrientTableProps = {
+  nutrition: Nutrient[];
+};
+
+export const NutrientTable = ({ nutrition }: NutrientTableProps) => {
+  return (
+    <ul className="w-full max-w-sm mx-auto">
+      {nutrition.map((nutrient, i) => (
+        <li
+          key={nutrient.name}
+          className={`flex justify-between items-center px-10 py-5 bg-${
+            i % 2 === 0 ? 'gray-50' : 'white'
+          }`}
+        >
```

Comment:
https://tailwindcss.com/docs/hover-focus-and-other-states#odd
```suggestion
      {nutrition.map((nutrient) => (
        <li
          key={nutrient.name}
          className="flex justify-between items-center px-10 py-5 bg-white even:bg-gray-100"
        >
```